fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -80,6 +80,13 @@ public class Column implements Writable, GsonPreProcessable, GsonPostProcessable
 
     @SerializedName(value = "name")
     private String name;
+
+    // If the logical name exists,
+    // the logical name will be used and converted to the physical name when the user uses it.
+    // Use physical name if not present
+    @SerializedName("logicalName")
+    private String logicalName;
```

Comment:
Better to add comments on the relationship between logicalName and name in front of this class.